File: CMS/src/app/shared/DanhMucLoaiHinh.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucLoaiHinh } from 'src/app/shared/DanhMucLoaiHinh.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucLoaiHinhService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucLoaiHinh";
    }
}

File: Service/Implement/DanhMucLoaiHinhService.cs
namespace Service.Implement
{
    public class DanhMucLoaiHinhService : BaseService<DanhMucLoaiHinh, IDanhMucLoaiHinhRepository>
    , IDanhMucLoaiHinhService
    {
    private readonly IDanhMucLoaiHinhRepository _DanhMucLoaiHinhRepository;
    public DanhMucLoaiHinhService(IDanhMucLoaiHinhRepository DanhMucLoaiHinhRepository) : base(DanhMucLoaiHinhRepository)
    {
    _DanhMucLoaiHinhRepository = DanhMucLoaiHinhRepository;
    }
    }
    }

